src/main/java/com/gooddata/warehouse/Warehouse.java

```diff
@@ -92,6 +95,13 @@ public String getDescription() {
         return description;
     }
 
+    /**
+     * Get jdbc connection string. Works only on Warehouse loaded from API (using WarehouseService).
```

Comment:
the phrase "Works only on Warehouse loaded from API (using WarehouseService)" seems a bit redundant -- it is by nature for other fields mainly id or uri...